Nodejs/Product/Npm/SPI/NpmCommander.cs

```diff
@@ -178,24 +174,16 @@ private class NmpCommandRunner : IDisposable {
             return type;
         }
 
-        private async Task<bool> UninstallPackageAsync(string packageName, bool global) {
```

Comment:
we may want to consider keeping the global behavior in NpmCommander, as there's a clear enough abstraction. I don't have a strong preference either way.